src/components/shared/high/AvatarGroup.js

```diff
@@ -38,6 +50,11 @@ const AvatarGroup = ({
 			<Stack isInline {...props}>
 				{clones}
 			</Stack>
+			{showNames && <Names>{names.map((m, i)=>{
```

Comment:
In the future, there may potentially be many, many avatars, so the `max` property was implemented to account for a huge number of names. We should probably limit the max number of names with `max`, too.

Reply:
![image](https://user-images.githubusercontent.com/46811144/80258281-b66ad780-8637-11ea-8800-6364b6b767c2.png)
Hi, thanks for the suggestion, I handled this inside map function: if current index > max, map will return null, which, by default of JSX, will not show on the screen. Is this way okay or we'd better to cut the array to fixed size(max)